Changes

```diff
@@ -1,5 +1,8 @@
 Revision history for Perl module Alien::Base.
 
+1.08
```

Comment:
Same here for the version.  I appreciate adding the bullet point, but no version.

Reply:
Done